src/control/drpc/modules.go

```diff
@@ -83,6 +84,8 @@ const (
 	ModuleSrv ModuleID = C.DRPC_MODULE_SRV
 	// ModuleSecurity is the dRPC module for security tasks in DAOS server
 	ModuleSecurity ModuleID = C.DRPC_MODULE_SEC
+	// GetAttachInfo protocol version
+	ProtocolVersion = C.GET_ATTACH_INFO_PROTOCOL_VERSION
```

Comment:
Wondering if it might make sense to have the ProtocolVersion apply to the entire dRPC protocol, rather than considering it only an issue with GetAttachInfo. I suppose the end result is the same, but I'm just thinking from a structural level.

Also, a minor comment: I would suggest not defining ProtocolVersion in this block just because these are all module IDs here. Maybe at the top of the file. I can see protocol version as metadata, in a sense, for dRPC.

Reply:
I've moved the definition and renamed it to DRPC_PROTOCOL_VERSION as you requested.